internal/api/device_settings_service.go

```diff
@@ -25,14 +25,7 @@ func NewDeviceSettingsConfigService(logger *zerolog.Logger, dbs func() *db.Reade
 func (s *DeviceSettingsConfigService) CreateDeviceSettings(ctx context.Context, in *grpc.UpdateDeviceSettingsRequest) (*emptypb.Empty, error) {
 	service := commands.NewCreateDeviceSettingsCommandHandler(s.dbs)
 	_, err := service.Execute(ctx, &commands.CreateDeviceSettingsCommandRequest{
-		TemplateName:                           in.DeviceSettings.TemplateName,
-		BatteryCriticalLevelVoltage:            in.DeviceSettings.BatteryCriticalLevelVoltage,
-		SafetyCutOutVoltage:                    in.DeviceSettings.SafetyCutOutVoltage,
-		SleepTimerEventDrivenInterval:          in.DeviceSettings.SleepTimerEventDrivenInterval,
-		SleepTimerEventDrivenPeriod:            in.DeviceSettings.SleepTimerEventDrivenPeriod,
-		SleepTimerInactivityAfterSleepInterval: in.DeviceSettings.SleepTimerInactivityAfterSleepInterval,
-		SleepTimerInactivityFallbackInterval:   in.DeviceSettings.SleepTimerInactivityFallbackInterval,
-		WakeTriggerVoltageLevel:                in.DeviceSettings.WakeTriggerVoltageLevel,
+		TemplateName: in.DeviceSettings.TemplateName,
```

Comment:
I feel like we need a property for the JSON contents for the configuration settings. I think it could just be a string property, or byte array that represents the JSON config